elliptic-curve/src/hash2curve/osswu.rs

```diff
@@ -11,7 +11,7 @@ where
     F: Field,
 {
     /// The first constant term
-    pub c1: [u64; 4],
+    pub c1: &'static [u64],
```

Comment:
Could this be represented by something like [`Curve::UInt`](https://docs.rs/elliptic-curve/latest/elliptic_curve/trait.Curve.html#associatedtype.UInt) instead?

That can parse a hex string to an word-sized number of limbs.

Reply:
I think that would work and would also be much better. The only problem is that `OsswuMap` is implemented over `FieldElement`, so I'm not sure how exactly to proceed.

As far as I'm aware there is not link between `Curve` and it's `FieldElement`, so I can't depend on that either.

I could depend on `GroupDigest` instead directly on `FieldElement`, which establishes exactly that link.
Alternatively I could just add an associated type in `OsswuMap` that holds the curve.

Reply:
Aah, that's unfortunate, although short of the `ff` crate adopting something like `crypto-bigint` I'm not sure what better representation they could use

Reply:
Alright, gonna mark this as resolved then, feel free to open it again if you can think of a solution.

Reply:
FWIW I opened an upstream issue. Perhaps we could get a change to `ff` in before the next breaking change to the `elliptic-curve` crate:

https://github.com/zkcrypto/ff/issues/85

Reply:
And I guess this restriction still exists in `ff` v0.13.0 so I'll call this resolved for now.